src/core/mac/link_raw.hpp

```diff
@@ -237,13 +242,31 @@ class LinkRaw
      */
     otError SetExtAddress(const otExtAddress &aExtAddress);
 
+    /**
+     * This method gets the transmit frame.
+     *
+     * @returns A pointer to the transmit frame.
+     *
+     */
+    otRadioFrame *GetTransmitFrame(void) { return mTransmitFrame; }
+
 private:
-    otInstance &mInstance;
+    enum
+    {
+        kOperationTransmitData = 1 << 0,
+    };
+
+    void        TransmitNow(void);
+    void        StartTransmit(void);
+    static void HandleOperationTask(Tasklet &aTasklet);
+    void        HandleOperationTask();
+
+    Tasklet  mOperationTask;
+    uint16_t mPendingOperations;
 
 #if OPENTHREAD_LINKRAW_TIMER_REQUIRED
 
-    enum TimerReason
-    {
+    enum TimerReason{
```

Comment:
Is this change intentional?

Reply:
No, it's caused by my local clang-format-6.0. I have moved the macro location and it magically get right now.